components/Coverups/CoverupsDescription.jsx

```diff
@@ -0,0 +1,67 @@
+import styled from "@emotion/styled";
+import Image from "next/image";
+
+const CoverupsDescription = () => {
+  return (
+    <DescriptionContainer>
```

Comment:
Если нет ничего между объявление и return можем использовать стрелочную фукнцию

```suggestion
const CoverupsDescription = () => (
    <DescriptionContainer>
```

Reply:
Поправил